deploy/ui-plugin/configmap.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```

Comment:
can we move all this `deploy/openshift/ui-plugin` ? at the end this only works for openshift